lidar_lite_v3hp/src/lidar_litev3hp_wrapper.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+#include "lidar_litev3hp_wrapper.h"
+
+LidarLiteNode::LidarLiteNode()
+  {
+    status_.trailer_angle_sensor= false;
+    timer_ = nh_.createTimer(ros::Duration(1.0), &LidarLiteNode::updateLidarStatus,this);
+    sub_1_.subscribe(nh_, "range_sensor1/range", 1);
+    sub_2_.subscribe(nh_, "range_sensor2/range", 1);
+    sync_.reset(new Sync(MySyncPolicy(10), sub_1_, sub_2_));
+    sync_->registerCallback(boost::bind(&LidarLiteNode::sensorCallback, this, _1, _2));
+    pub_ang_=nh_.advertise<std_msgs::Float64>("trailer_angle",10);
+    sub_alert_=nh_.subscribe("system_alert",10,&LidarLiteNode::alertCallback,this);
+    pub_status_=nh_.advertise<cav_msgs::DriverStatus>("driver_discovery", 10);
+  }
+
+   void LidarLiteNode::sensorCallback(const sensor_msgs::RangeConstPtr &sensor_inp1, const sensor_msgs::RangeConstPtr &sensor_inp2)
+  {
+    status_.trailer_angle_sensor= true;
+    status_lidar_=cav_msgs::DriverStatus::OPERATIONAL;
+
+    if(status_lidar_==cav_msgs::DriverStatus::OPERATIONAL)
+    {
+      status_.status=cav_msgs::DriverStatus::OPERATIONAL;
+      last_update_time_=ros::Time::now();
+    }
+
+    double sensor_distance; //Distance between sensor
+    nh_.getParam("/lidar_litev3hp/distance_between_two_sensor",sensor_distance);
```

Comment:
This should be "~/distance_between_two_sensor"
as this was loaded as a private parameter